CONTRIBUTING.rst

```diff
@@ -0,0 +1,74 @@
+Contribution Guidelines
+=======================
+
+We use `Github projects`_ to organize our ticket workflow.
+If you want to lend a hand, check out the current project and choose one of the tickets from the issues!
+If there's `a particular issue`_ you would like to work on and it isn't already assigned, leave a comment on that issue indicating your desire to work on it.
+Then, start working!
+
+Start Developing
+----------------
+
+To get started with your contribution, fork this project into your own GitHub profile and clone that forked repository to your machine.
+As soon as you finish cloning, navigate into the new directory and set the original project repository as the remote name `upstream`.
+
+.. code-block::
+
+    $ cd django-graph-api
+    $ git remote add upstream https://github.com/melinath/django-graph-api.git
+
+We're using `pipenv`_ as the package and environment manager for this project.
+If you don't yet have it, check the link above for instructions on how to get it on your machine.
+
+Create a ``pipenv`` virtual environment using Python 3.6.
+**Note: any code that you write should be compatible with Python 2.7, but we recommend that you develop in Python 3.6.**
+``pipenv`` install the ``requirements.txt``, then ``pipenv`` the requirements for testing.
+
+.. code-block::
+
+    $ pipenv install --python 3.6
```

Comment:
I think install can be removed from here if the following lines are executed, right?

Reply:
The point of this line is to specify that you're developing in Python 3.6 specifically. Otherwise, `pipenv` goes with whatever your default Python is (and on my machine that's Python 2.7)

Reply:
Oh I just meant `pipenv --python 3.6` would do that without including the install command since the next two lines do the installs.